db/migrate/20220209211536_add_ap_footer_fields_to_projects.rb

```diff
@@ -0,0 +1,21 @@
+class AddApFooterFieldsToProjects < ActiveRecord::Migration
```

Comment:
If you give this migration the same file name as the one in the new-sections (LARA 2) branch then the migration should not run again when new-sections is merged in.